src/util/sortingTypeConstants.js

```diff
@@ -0,0 +1,9 @@
+export const sortingTypeForSearch = {
```

Comment:
I think it's better to place all related helpers and constants to module folder(of course if they are not used somewhere else)

Reply:
done